homeassistant/components/brother/__init__.py

```diff
@@ -53,6 +52,6 @@ async def async_unload_entry(hass: HomeAssistant, entry: BrotherConfigEntry) ->
     ]
     # We only want to remove the SNMP engine when unloading the last config entry
     if unload_ok and len(loaded_entries) == 1:
-        hass.data[DOMAIN].pop(SNMP)
+        hass.data.pop(SNMP_ENGINE)
```

Comment:
Just thinking, do we really what to pop the `SNMP_ENGINE` here? That would prevent us from using it in another integration. Additionally (if we keep it here), do we need to call `lcd.unconfigure` before it like we do for shutdown?

Reply:
Good point, my proposition is... currently only `brother` uses `snmp_engine` so let's add `unconfigure` here before `pop`. Once I manage to use `snmp_engine` for `snmp` integration, we will remove this piece of code.